config/config.go

```diff
@@ -0,0 +1,41 @@
+package config
+
+import (
+	"embed"
+	"sync"
+)
+
+type ProjectConfig struct {
```

Comment:
why not just `Config`?

Reply:
> renaming this type "ProjectConfig" to "Config"	conflicts with func in same blockrenaming this type "ProjectConfig" to "Config"	conflicts with func in same blockrenaming this type "ProjectConfig" to "Config"	conflicts with func in same block

plus the package name is the same 

what about `Cfg`?